src/domain/tradingview/useTVDatafeed.ts

```diff
@@ -0,0 +1,91 @@
+import { getTokens } from "config/tokens";
+import { timezoneOffset } from "domain/prices";
+import { useChainId } from "lib/chains";
+import { useMemo, useRef } from "react";
+import { getHistoryBars, getLiveBar } from "./requests";
+import { supportedResolutions } from "./utils";
+
+const configurationData = {
+  supported_resolutions: Object.keys(supportedResolutions),
+  supports_marks: false,
+  supports_timescale_marks: false,
+  supports_time: true,
+  reset_cache_timeout: 100,
+};
+
+export default function useTVDatafeed() {
+  const { chainId } = useChainId();
+  const intervalRef = useRef<ReturnType<typeof setInterval> | null>();
+  const activeTicker = useRef<string | null>();
+  return useMemo(() => {
+    return {
+      onReady: (callback) => {
+        setTimeout(() => callback(configurationData));
+      },
+      resolveSymbol: async (symbolName, onSymbolResolvedCallback) => {
+        const stableTokens = getTokens(chainId)
+          .filter((t) => t.isStable)
+          .map((t) => t.symbol);
+        const symbolInfo = {
+          name: symbolName,
+          type: "crypto",
+          description: symbolName + " / USD",
+          ticker: symbolName,
```

Comment:
is there a public documentation for widget? would be good to leave link here so it's easier to understand what this config fields mean

Reply:
Sorry the repo that contains the docs is not public.